provisioning/templates-api/common.sh

```diff
@@ -38,6 +38,7 @@ helm upgrade \
   --install templates-api-etcd bitnami/etcd \
   --version "$ETCD_HELM_CHART_VERSION" \
   --values ./kubernetes/deploy/etcd/values.yaml \
+  --values ./kubernetes/deploy/etcd/values_templates.yaml \
```

Comment:
Tu by malo stacit `--set "persistence.storageClass=$ETCD_STORAGE_CLASS_NAME"`, zbytocne to komplikovat cez `envsubstr`.

Reply:
`ETCD_STORAGE_CLASS_NAME` by si mal potom pridat medzi `required envs` v `common.sh`.

https://github.com/keboola/keboola-as-code/blob/23a34f89ca49a1211865b6fb272dd898b0aaad47/provisioning/templates-api/common.sh#L10-L17

Reply:
Jo to je taky varianta, ale pro storage už tam definujete custom value přes file (u buffer API). Tak mi přišlo divné to definovat na dvou místech. Ale klidně to tak udělám.

Reply:
su to samostatne hodnoty, tak v tom nevidim problem, ... hlavne to bude o dost jednoduchsie, pri rovnakom vysledku

Reply:
Ok, upravím